Beam/resources/assets/js/remplib.js

```diff
@@ -48,6 +48,16 @@ remplib = typeof(remplib) === 'undefined' ? {} : remplib;
 
         timeSpentActive: false,
 
+        trackedProgress: [],
+
+        trackedArticle: null,
```

Comment:
Whole article-related tracking should not proceed if article element is not initialized properly. I got lot of errors all over the place when `trackedArticle` was undefined.